Packs/PICUS/Integrations/PICUS/PICUS.py

```diff
@@ -0,0 +1,901 @@
+        'Contents': results,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': hr
+    }
+
+
+'''Response:
+{
+  "data": {
+    "page_number": 2,
+    "sigma_rules": [
+      {
+        "actions": [
+          {
+            "description": "string",
+            "id": 0,
+            "name": "string"
+          }
+        ],
+        "content": "title: Gathering PnP Device List Information via powershell status: experimental...",
+        "id": 0
+      }
+    ],
+    "size": 5,
+    "total_count": 548
+  },
+  "success": true
+}
+'''
+
+
+def vectorList(requestContent):
+    SERVER = demisto.params().get('server')[:-1] if str(demisto.params().get('server')).endswith('/') \
+        else demisto.params().get('server')
+    endpoint = "/user-api/v1/vectors/list"
+
+    accessToken = "Bearer " + requestContent
+
+    headers = {"X-Api-Token": accessToken}
+
+    url = SERVER + endpoint
+    add_user_details = bool(demisto.args().get('add_user_details'))
+    page = int(demisto.args().get('page'))
+    size = int(demisto.args().get('size'))
+    data = {'add_user_details': add_user_details,
+            'page': page,
+            'size': size
+            }
+
+    req = requests.post(url, headers=headers, data=json.dumps(data), verify=VERIFY_SSL)
+    parsed = json.loads(req.content)
+
+    results = parsed['data']['vectors'] if 'data' in parsed else ''
+
+    header = ['name', 'description', 'trusted', 'untrusted', 'is_disabled', 'type']
+    hr = tableToMarkdown('Picus Vector List', results, header, removeNull=True)
+
+    return {
+        'Type': entryTypes['note'],
+        'Contents': results,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': hr
+    }
+
+
+'''Response:
+{
+  "data": {
+    "page_number": 2,
+    "size": 5,
+    "total_count": 548,
+    "vectors": [
+      {
+        "description": "string",
+        "heartbeat_results": [
+          {
+            "is_successful": true,
+            "module": "PTS",
+            "result_time": "2018-08-21T13:00:33.590599407Z",
+            "variant": "HTTP"
+          }
+        ],
+        "is_disabled": true,
+        "name": "Email-Peer-2 - Email-Peer-1",
+        "trusted": "Windows10-Peer",
+        "type": "string",
+        "untrusted": "Network-Peer",
+        "users": [
+          {
+            "role": "admin",
+            "username": "oliver"
+          }
+        ]
+      }
+    ]
+  },
+  "success": true
+}
+'''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    ''' EXECUTION '''
+    #LOG('command is %s' % (demisto.command(), ))
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        LOG('Command being called is {command}'.format(command=demisto.command()))
+        if demisto.command() == 'Picus-GetAccessToken':
```

Comment:
Notice that the YML doesn't state that there is a command like that.
In case you would like to add it to the YML, use the name convention below.

```suggestion
        if demisto.command() == 'picus-get-access-token':
```